lib/Sema/SemaType.cpp

```diff
@@ -3522,7 +3522,7 @@ static void recordNullabilitySeen(Sema &S, SourceLocation loc) {
     S.Diag(fileNullability.PointerLoc, diag::warn_nullability_missing_array);
   } else {
     S.Diag(fileNullability.PointerLoc, diag::warn_nullability_missing)
-      << fileNullability.PointerKind;
+      << static_cast<unsigned>(fileNullability.PointerKind);
```

Comment:
This has been fixed upstream; I just hadn't cherry-picked it. Can you take the real commit instead?